src/test/java/org/opentripplanner/model/plan/legreference/LegReferenceSerializerTest.java

```diff
@@ -8,31 +8,33 @@
 
 class LegReferenceSerializerTest {
 
+  private static final FeedScopedId TRIP_ID = new FeedScopedId("F", "Trip");
+  private static final ServiceDate SERVICE_DATE = new ServiceDate(2022, 1, 31);
+  private static final int FROM_STOP_POS = 1;
+  private static final int TO_STOP_POS = 3;
+
   @Test
   void testScheduledTransitLegReferenceRoundTrip() {
-    var ref = new ScheduledTransitLegReference(
-      new FeedScopedId("Feed", "Trip"),
-      new ServiceDate(2022, 1, 31),
-      1,
-      3
-    );
+    var ref = new ScheduledTransitLegReference(TRIP_ID, SERVICE_DATE, 1, 3);
 
     var out = LegReferenceSerializer.encode(ref);
 
+    System.out.println(out);
```

Comment:
Is this necessary?

Reply:
I improved the test and removed the System.out.